.env.production

```diff
@@ -1 +1 @@
-NEXT_PUBLIC_BASE_API_URL='https://api.realdevsquad.com'
\ No newline at end of file
+NEXT_PUBLIC_BASE_API_URL=https://api.realdevsquad.com
```

Comment:
This minus sign indicates that there is no empty line
![image](https://user-images.githubusercontent.com/56217868/132951488-a9b26219-627d-4f94-9773-5a8b3f9327d2.png)

Reply:
From file->preferences->settings, i've enabled "insert final newline".